test/e2e/set/scyllacluster/scyllacluster_shardawareness.go

```diff
@@ -69,57 +66,131 @@ var _ = g.Describe("ScyllaCluster", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(hosts).To(o.HaveLen(1))
 
-		connections := make(map[uint16]string)
-		var connectionsMut sync.Mutex
-
-		clusterConfig := gocql.NewCluster(hosts...)
-		clusterConfig.Dialer = DialerFunc(func(ctx context.Context, network, addr string) (net.Conn, error) {
-			sourcePort := gocql.ScyllaGetSourcePort(ctx)
-			localAddr, err := net.ResolveTCPAddr(network, fmt.Sprintf(":%d", sourcePort))
-			if err != nil {
-				return nil, err
-			}
-
-			framework.Infof("Connecting to %s using %d source port", addr, sourcePort)
-			connectionsMut.Lock()
-			connections[sourcePort] = addr
-			connectionsMut.Unlock()
+		clientPod := &corev1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "client",
+			},
+			Spec: corev1.PodSpec{
+				Containers: []corev1.Container{
+					{
+						Name:  "client",
+						Image: configassests.Project.OperatorTests.NodeSetupImage,
+						Command: []string{
+							"/bin/sh",
+							"-c",
+							"sleep 3600",
```

Comment:
nit: this can call sleep directly, possibly without the termination period

Reply:
`infinity`?

Reply:
done